src/index.ts

```diff
@@ -127,7 +180,8 @@ function getEmit(
     // Emit Javascript
     const output = instance.languageService.getEmitOutput(filePath);
 
-    loader.clearDependencies();
+    // Remove original dependency on resourcePath and replace with rawFilePath
```

Comment:
NB. I have just checked the code in [webpack/loader-runner](https://github.com/webpack/loader-runner) again, and my assumption that, if ts-loader is the only loader, `resourcePath` will be the only dependency, is correct.
So the only reason, I can think of, for the original call to `clearDependencies` is to replace `resourcePath` with the same path, but normalized.